rules/inclusive-gateway.js

```diff
@@ -0,0 +1,52 @@
+const { is } = require('bpmnlint-utils');
+
+const { hasProperties, ERROR_TYPES } = require('./utils/element');
+
+const { reportErrors } = require('./utils/reporter');
+
+module.exports = function() {
+  function check(node, reporter) {
+    if (!is(node, 'bpmn:InclusiveGateway')) {
+      return;
+    }
+
+    const incoming = node.get('incoming');
+
+    if (incoming && incoming.length > 1) {
```

Comment:
Note that associations can also be members of `incoming`. We should care only about sequence flows.

Reply:
`incoming` and `outgoing` can be `bpmn:SequenceFlow` elements (cf. https://github.com/bpmn-io/bpmn-moddle/blob/v7.1.3/resources/bpmn/json/bpmn.json#L1726).

Reply:
Oh, OK. I confused `bpmn:incoming` with diagram-js incoming.